cmd/pow/cmd/root.go

```diff
@@ -56,6 +53,8 @@ func init() {
 	rootCmd.Flags().BoolP("version", "v", false, "display version information for pow and the connected server")
 	rootCmd.PersistentFlags().String("serverAddress", "127.0.0.1:5002", "address of the powergate service api")
 	rootCmd.PersistentFlags().StringP("token", "t", "", "user auth token")
+
+	rootCmd.AddCommand(data.Cmd)
```

Comment:
This nested package design make it more natural for parent commands to attach to their child commands, which seems like the right relationship anyway.